posthog/hogql/property.py

```diff
@@ -367,8 +367,6 @@ def action_to_expr(action: Action) -> ast.Expr:
         if step.event == AUTOCAPTURE_EVENT:
             if step.selector:
                 exprs.append(selector_to_expr(step.selector))
-            if step.tag_name is not None:
```

Comment:
We still use this in the backend `format_action_filter()` and plugin server `elementToSelector()`, so we'd have to remove that for completeness. But I'm a bit worried about unexpected changes of results for users, so might be safer to only stop action creation from including `tag_name`. But I thought I tackled this exact problem in #15793! Did we miss a spot back then?

Reply:
Hmm, this might not be the best fix then, I'd need to revisit. 
This is the support request: https://posthoghelp.zendesk.com/agent/tickets/12658
I had forgotten all about that PR of yours by now. This action might have been earlier, didn't check.

Reply:
I reverted this.